larpandoracontent/LArControlFlow/StitchingCosmicRayMergingTool.cc

```diff
@@ -594,10 +660,36 @@ void StitchingCosmicRayMergingTool::CalculateX0(const PfoToLArTPCMap &pfoToLArTP
                 LArStitchingHelper::GetClosestVertices(*pLArTPC1, *pLArTPC2, pointingCluster1, pointingCluster2,
                     pointingVertex1, pointingVertex2);
 
+                PfoToPointingVertexMap::iterator pfoToPointingVertexMapIter1(pfoToPointingVertexMap.find(pPfo1));
+                if (pfoToPointingVertexMapIter1 == pfoToPointingVertexMap.end())
+                {
+                    pfoToPointingVertexMap.insert(std::make_pair(pPfo1, pointingVertex1));
```

Comment:
Cosmetic: For a map of type `MyMap`, I tend to use:
`myMap.insert(MyMap::value_type(key, value));`

The insertion also returns a pair, with the first element being an iterator to the inserted value (or existing value), and the second element being a Boolean indicating whether a new element was inserted (true), or whether an equivalent key already existed (false). I generally check the Boolean, but here it's not useful, as you know the map didn't already contain a key matching `pPfo1`, so in this case, I normally indicate that I'm throwing away the return value:
`(void) myMap.insert(MyMap::value_type(key, value));`